docs/DeveloperGuide.md

```diff
@@ -373,32 +373,33 @@ Below is a sequence diagram that illustrates how a user adds new tasks into the
   creating tasks may be forced to key in random content to add tasks.
 
 #### Constraints:
-**Title must be unique:** </br>
+**Title must be unique:**  
 We felt that the title should be unique as it improves organisation and visual clarity for the user. By mandating unique 
 titles, we encourage users to be specific in the title(purpose) of the task (e.g they will set title as 
 "Complete slides for Mr X" rather than "Complete Slides"), which will benefit them greatly, as they will be 
-able to clearly distinguish the purpose of each task just by looking at the title. </br>
+able to clearly distinguish the purpose of each task just by looking at the title.  
 Suppose that the title was not unique. Users might have many tasks with the same title, which would impair their ability
 to distinguish between the tasks unless they read each of the task content individually. It would also impair visual
 clarity when searching for tasks, as tasks with similar titles might clutter up the GUI.
 Hence, our approach in mandating unique titles are geared towards improving organisation and visual clarity for users in 
 both the short and long term.
 
 ### Deleting a task
-Syntax: `deletetask INDEX` </br>
+Syntax: `deletetask INDEX`  
 Purpose: Allows users to delete the task at the specified index in the OfficeConnectModel.
 
 #### Implementation
 The implementation of this feature is supported by `ListTaskCommand`, `DeleteTaskCommand` and `DeleteTaskCommandParser`.
-Below are the steps required to delete a task in the OfficeConnectModel. </br>
+Below are the steps required to delete a task in the OfficeConnectModel.  
 
 Step 1: User keys in `listtask`, which will display the index of all tasks. 
 The user can thus obtain the index of the task that they want to delete.
 
 Step 2: User keys in `deletetask INDEX` to delete the task at the specified index. 
 If the index is invalid, an error will be thrown. 
 
-Below is an activity diagram showcasing the 2 steps: </br> 
+Below is an activity diagram showcasing the 2 steps:
+
 ![DeleteTaskActivityDiagram](images/DeleteTaskActivityDiagram.png)
```

Comment:
Your Invalid INDEX has a typo it should be app/ officeconnect throws an error, not OfficeConnectModel

Reply:
great catch!